api/cinemaHall/CinemaHall.js

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable no-await-in-loop */
 import mongoose from 'mongoose';
 import Seat from '../seat/Seat.js';
+import Screening from '../screening/Screening.js';
 
 const { Schema } = mongoose;
 
 const cinemaHallSchema = new Schema({
-  name: { type: Number, required: true },
+  name: { type: String, required: true, unique: true },
```

Comment:
The name has to be unique within one cinema, so if we have more cinemas, the names can be repeated. What we need is a check, if in this cinema already exists another hall with this name.